packages/cli/src/commands/deploy/args.ts

```diff
@@ -42,6 +42,7 @@ export const help = () => `
 
     -h, --help                     Output usage information
     -v, --version                  Output the version number
+    --cwd                          Current working directory
```

Comment:
This seems to be where top-level args are described in `help`. Is this the right place?

Reply:
I think so.

We'll also want to update the global config docs once this lands https://vercel.com/docs/cli#introduction/global-options